api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -120,6 +122,15 @@
 	        List<ConceptDatatype> excludeDatatypes, Concept answersToConcept, Integer start, Integer size)
 	        throws DAOException;
 	
+	/**
+	 * Return the number of concepts matching a search phrase and the specified arguments
+	 *
+	 * @param phrase matched to the start of any word in any of the names of a concept
+	 * @return the number of concepts matching the given search phrase
+	 * @throws APIException
+	 * @since 1.8
+	 * @should return a count of unique concepts
+	 */
```

Comment:
Since this method is well documented in the `ConceptService` interface, don't you wanna reference to that documentation with the help of `@see` javadoc attribute just like its done in other places in this class?